config/locales/en/course/course_users.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  course:
+    course_users:
+      show:
+        header: 'Profile'
+        role: 'Role: %{role}'
+        experience_points_earned_html: 'Experience points earned: %{points}'
+        achievement_count: 'Achievement count: %{count}'
```

Comment:
maybe it's more natural to say '%{count} achievements' with differing keys for one and zero and many. Rails has good support for natural pluralisation.